pages/index.js

```diff
@@ -47,5 +48,13 @@ export default () => (
         { label: 'jenkins sshd', path: 'Core/job/sshd-module/job/master' }
       ]}
     />
+
+    <ElasticsearchErrorHitCount
+      title='Log Hits'
+      url='http://localhost:3001/http://localhost:9200'
```

Comment:
We have public urls for demo / testing (with crossorigin.me). It would be great if you find an elasticsearch demo project and use it 👍🏻